### PR TITLE
Allow example, thumbnail generation to happen async with nx dev

### PR DESCRIPTION
**Pull request by jacobp100**

### CODE REVIEW

File: packages/all/project.json [hunk 1]
```diff
@@ -70,7 +70,6 @@
     },
     "dev": {
       "executor": "nx:run-commands",
-      "dependsOn": ["dev:setup"],
```
Comment (alantreadway) [inline packages/all/project.json]: Revert?

**Review by alantreadway — APPROVED** [review]
LGTM, except I'm still seeing the `project.json` change?

File: packages/all/project.json (hunk 1 above)
Comment (jacobp100) [reply packages/all/project.json]: Not sure what happened - fixed

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**